components/NewsletterHome.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useRuntimeConfig } from '#app'
+import axios from 'axios'
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+
+const config = useRuntimeConfig()
+
+const props = defineProps({
+  showNoThanks: {
+    type: Boolean,
+    default: false,
+  },
+  altDesign: {
+    type: Boolean,
+    default: false,
+  },
+  submitButtonText: {
+    type: String,
+    default: 'Sign up',
+  },
+  submitButtonIcon: {
+    type: String,
+    //default: 'pi-arrow-right',
+    default: null,
+  },
+})
+
+const emit = defineEmits<{
+  (e: 'submitEventEmit', value: any): void
+}>()
+
+const showComp = ref(true)
```

Comment:
Can we spell out the word "Component" here. It's not immediately obvious if this is referring to component or composition or completion or something else here.